libs/watt/package/segmented-buttons/watt-segmented-buttons.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+//#region License
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//#endregion
+import { render, screen } from '@testing-library/angular';
+import { WattSegmentedButtonsComponent } from './watt-segmented-buttons.component';
+
+describe.skip(WattSegmentedButtonsComponent, () => {
+  it('renders', async () => {
+    await render(WattSegmentedButtonsComponent);
+
+    expect(screen.queryAllByRole('slider')).toHaveLength(2);
+  });
+});
```

Comment:
i don't believe this test is executed (it has `.skip`). also the `slider` assertion seems wrong, unless the `MatButtonToogle` is somehow set up like that. either fix the test or remove it if it doesn't provide any value.

Reply:
Sorry - det var da vidst en copy paste fejl - kigger på det!